lib/applications.js

```diff
@@ -17,7 +18,7 @@ const errors = require('./errors');
 //
 class Application {
   constructor(name, api, version) {
-    [this.name, this.version] = common.rsplit(name, '@');
+    [this.name, this.version] = common.rsection(name, '@');
```

Comment:
I agree, we need full `rsplit` so I added issue here: https://github.com/metarhia/common/issues/91 but I think in this place `rsection` is ok because it is simpler and in multiple other cases `rsection` will be optimal.